docs/03-github/01-getting-started.mdx

```diff
@@ -219,11 +219,13 @@ jobs:
           - Android # Build an Android player.
           - WebGL # WebGL.
     steps:
-      - uses: actions/checkout@v2
+      - if: matrix.targetPlatform == 'Android'
```

Comment:
This should probably come after test-runner.

Reply:
Moved